subprojects/plugins/src/integTest/resources/org/gradle/groovy/compile/BasicGroovyCompilerIntegrationSpec/compileWithAnnotationProcessor/settings.gradle

```diff
@@ -0,0 +1,7 @@
+include ':'
+include ':SimpleAnnotationProcessor'
+// Integration tests don't copy directories that aren't named after a test to build/tmp/test files, where these build scripts are actually run from.
```

Comment:
Would you mind inlining this stuff? By this, I mean having the tests write this stuff to disk programmatically. It's easier to compose that way and it's also easier to see what the test is actually testing at a glance.